meta/applicator.json

```diff
@@ -4,47 +4,47 @@
     "$vocabulary": {
         "https://json-schema.org/draft/2019-09/vocab/applicator": true
     },
-    "$recursiveAnchor": true,
+    "$dynamicAnchor": "meta",
```

Comment:
I'm not sure if "meta" is the best name for this, but I am flailing at coming up with a better alternative.  "subschema" maybe?

Reply:
I think this is named such because it's in the "meta" applicator schema.
OK to close this?

Reply:
It's called "meta" because it's the anchor for metaschemas.  It must be the same name in all of them.  It needs to not collide with any other casually likely name that a schema author might use elsewhere.  What else would you call it?

Reply:
ah right, of course. I'm sure I read this and appreciated that reason for naming at some point in the past... =]

Reply:
I'm going to resolve this. I don't think there's any action here.

Reply:
ok. "metaschema" might be more clear than just "meta", but I do get that it's just a name, and naming things is subjective. :)